src/alire/alire-platforms.ads

```diff
@@ -40,6 +40,7 @@ package Alire.Platforms with Preelaborate is
                           Rhel, -- RedHat Enterprise Linux
                           Centos,
                           Fedora,
+                          MacOS,
```

Comment:
This entry should be `Homebew` and not `MacOS`, for the same reasons that we have `Msys2` and not `Windows`:
 - You can use Alire on MacOS without Homebew
 - You can have other system package managers on MacOS (Macports)

Reply:
OK

Reply:
Homebrew also supports Windows 10 subsystem and Linux. You might need to create a package matrix or perhaps another environment variable for the parent package manager, i.e. how was Alire installed regardless of platform. This might be a consideration if, for instance, some operation might need the information.

Reply:
> Homebrew also supports Windows 10 subsystem and Linux. You might need to create a package matrix or perhaps another environment variable for the parent package manager, i.e. how was Alire installed regardless of platform. This might be a consideration if, for instance, some operation might need the information.

When you say "You might need to create ..." I hope you don't mean me? This sounds way outside my comfort zone

Reply:
> When you say "You might need to create ..." I hope you don't mean me? This sounds way outside my comfort zone

I guess it was said as hypothetical future feature. In any case, Alire currently only supports one package manager per distro. But, certainly, one can use combos [os x distro] in the externals, although the need has not yet arisen.

Which makes me think that, as we have `macOS` as the OS, maybe it's more appropriate to use `homebrew` as the distro just like `msys2` is used on Windows. This would enable using other package manager on macOS if someone wants to come up with it.

Reply:
Yes, that’s how I’m approaching it. I’m tidying up the update to this PR.